core/arch/arm/kernel/unwind_private.h

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+#include <malloc.h>
+#include <stddef.h>
+#include <util.h>
+
+#ifdef CFG_UNWIND
+static inline void *unw_grow(void *p, size_t *cur_size, size_t new_size)
+{
+	if (*cur_size >= new_size)
+		return p;
+
+	size_t rounded_size = ROUNDUP(new_size, 16 * sizeof(vaddr_t));
+
+	void *tmp = realloc(p, rounded_size);
```

Comment:
minor: add en empty line. maybe remove the one between the 2 local variables definition.

Reply:
Yep